FluentTerminal.App.Services/Implementation/SettingsService.cs

```diff
@@ -52,10 +52,16 @@ public SettingsService(IDefaultValueProvider defaultValueProvider, ApplicationDa
 
         public event EventHandler KeyBindingsChanged;
 
+        /// <summary>
+        /// Indicate that a change has occurred to the shell profile collection. If a profile was added, the furst tuple value will be true, and if a profile was removed it will be false.
+        /// </summary>
+        public event EventHandler<Tuple<bool, ShellProfile>> ShellProfileCollectionChanged;
```

Comment:
Why don't we split it into ShellProfileAdded and ShellProfileDeleted? Would be more expressive